cf/data/data.py

```diff
@@ -11552,6 +11773,21 @@ def ones(
         dx = da.ones(shape, dtype=dtype, chunks=chunks)
         return cls(dx, units=units, calendar=calendar)
 
+    #    def optimize_graph(self):
+    #        """Remove unnecessary tasks from the dask graph in-place.
+    #
+    #        .. versionadded:: 3.15.1
+    #
+    #        .. seealso:: `todict`
+    #
+    #        """
+    #        d = _inplace_enabled_define_and_cleanup(self)
+    #        dx = d.to_dask_array()
+    #        dsk = collections_to_dsk((dx,), optimize_graph=True)
+    #        dx = da.Array(dsk, name=dx.name, chunks=dx.chunks, dtype=dx.dtype)
+    #        d._set_dask(dx, clear=_NONE)
+    #        return d
+
```

Comment:
In line with removing dead code, unless there's a specific reason to keep it in (but git always remembers anyway!):

```suggestion
```